parsing_engines/mappings.py

```diff
@@ -1,37 +1,31 @@
-COLUMN_NAME_MAPPING = {
+"""
+TODO: Mappings can all be defined here! No need for separate files. 
+"""
+
+GENERIC_COLUMN_MAPPING = {
     "time": "time/s",
     "Time": "time/s",
-    "time/s": "time/s",
-    "mode": "mode",
-    "ox/red": "ox/red",
-    "error": "error",
-    "control changes": "control changes",
-    "Ns changes": "Ns changes",
     "counter inc.": "counter inc",
-    "Ns": "Ns",
-    "I Range": "I Range",
-    "control/V/mA": "control/V/mA",
-    "Ecell/V": "Ecell/V",
-    "I/mA": "I/mA",
     "dq/mA.h": "dq/mA_h",
     "(Q-Qo)/mA_h": "(Q-Qo)/mA.h",
     "Energy/W_h": "Energy/W.h",
-    "Analog IN 1/V": "Analog IN 1/V",
-    "Analog OUT/V": "Analog OUT/V",
     "Energy charge/W_h": "Energy charge/W.h",
     "Energy discharge/W_h": "Energy discharge/W.h",
-    "Capacitance charge/µF": "Capacitance charge/µF",
-    "Capacitance discharge/µF": "Capacitance discharge/µF",
-    "x": "x",
     "Q discharge/mA_h": "Q discharge/mA.h",
     "Q charge/mA_h": "Q charge/mA.h",
     "Capacity/mA_h": "Capacity/mA.h",
-    "Efficiency/%": "Efficiency/%",
-    "control/V": "control/V",
-    "control/mA": "control/mA",
-    "cycle number": "cycle number",
-    "P/W": "P/W",
-    "R/Ohm": "R/Ohm",
-    "Rec#": "Rec#",
     "RecNo": "Rec#",
 }
+
+BIOLOGIC_COLUMN_MAPPING = {
```

Comment:
Interesting that so many headings have been removed from the generic mapping while so few are added by the engine-specific ones. Were all these that have been removed just going unused?

Reply:
Yeah so actually these mappings were never properly implemented: The "column_name_mapping" in the parsing engine was _always_ set to `None`. This has been fixed now and these mappings are reorganised into `{"what could be in the file column header":"what is set as the column header in the Parser"}`, and the logic in the code should handle that accordingly. 

A lot of what's been deleted are instances where the key and val were equal, for which there is no real use.

Reply:
Ah ok cool, that makes sense. Cheers!